sdk/spring/azure-spring-cloud-starter-eventhubs/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 2.3.0-beta.1 (Unreleased)
-
+## 2.3.0 (2021-03-16)
+### New Features
+- Upgrade to `Spring Boot` [2.4.3](https://github.com/spring-projects/spring-boot/releases/tag/v2.4.3).
+- Upgrade to `azure-messaging-eventhubs` [5.6.0](https://github.com/azure-sdk/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md#560-2021-03-10).
```

Comment:
It's better to use `commit id` instead of `master`.

Reply:
we could keep it this way and if there's a change of folder or file name, let the link check detect this and we could change accordingly.